test/integration/test.conversation.js

```diff
@@ -594,6 +624,360 @@ describe('conversation_integration', function() {
+        }
+        assert.equal(result.entity, test_entities_update.entity);
+        done();
+      });
+    });
+  });
+
+  describe('createValue()', function() {
+    it('should create a value', function(done) {
+      const params = {
+        workspace_id: workspace1.workspace_id,
+        entity: test_entities_update.entity,
+        value: test_value.value,
+        synonyms: test_value.synonyms
+      };
+
+      conversation.createValue(params, function(err, result) {
+        if (err) {
+          return done(err);
+        }
+        assert.equal(result.value, test_value.value);
+        assert.equal(result.description, null);
+        done();
+      });
+    });
+  });
+
+  describe('getValues()', function() {
+    it('should get values of the entity', function(done) {
+      const params = {
+        workspace_id: workspace1.workspace_id,
+        entity: test_entities_update.entity,
+        export: true
+      };
+
+      conversation.getValues(params, function(err, result) {
+        if (err) {
+          return done(err);
+        }
+        assert.equal(result.values[1].value, test_value.value);
+        assert.equal(result.values[1].synonyms[0], test_value.synonyms[0]);
+        done();
+      });
+    });
+
+    it('should have pagination information', function(done) {
+      const params = {
+        workspace_id: workspace1.workspace_id,
+        entity: test_entities_update.entity,
+        export: true,
+        page_limit: 1,
+        include_count: true,
+        sort: 'value'
+      };
+
+      conversation.getValues(params, function(err, result) {
+        if (err) {
+          return done(err);
+        }
+        assert.equal(result.hasOwnProperty('pagination'), true);
+        done();
+      });
+    });
+  });
+
+  describe('getValue()', function() {
+    it('should get a value of the entity', function(done) {
+      const params = {
+        workspace_id: workspace1.workspace_id,
+        entity: test_entities_update.entity,
+        value: test_value.value
+      };
+
+      conversation.getValue(params, function(err, result) {
+        if (err) {
+          return done(err);
+        }
+        assert.equal(result.value, test_value.value);
+        assert.equal(result.description, null);
+        done();
+      });
+    });
+  });
+
+  describe('updateValue()', function() {
+    it('should update a value of the entity', function(done) {
+      const params = {
+        workspace_id: workspace1.workspace_id,
+        entity: test_entities_update.entity,
+        old_value: test_value.value,
+        value: test_value_update.value,
+        synonyms: test_value_update.synonyms
+      };
+
+      conversation.updateValue(params, function(err, result) {
+        if (err) {
+          return done(err);
+        }
+        assert.equal(result.value, test_value_update.value);
+        done();
+      });
+    });
+  });
+
+  describe('createSynonym()', function() {
+    it('should create a synonym', function(done) {
```

Comment:
When I run it locally, this test fails with

```json
{
  "error": {
    "error": {
      "text": [
        "Text can not be empty."
      ]
    }
  }
}
```

(Actually, the error it throws is `[object Object]` but I'll fix that part in master)

And, then this failure causes a cascade where nearly all of the other synonym tests fail.